main.go

```diff
@@ -27,6 +29,12 @@ const (
 func main() {
 	r := mux.NewRouter()
 
+	r.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		if err := renderer.HTML.Render(w, "home.html", nil); err != nil {
+			w.WriteHeader(500)
+			w.Write([]byte("Ocorreu um erro ao renderizar a página"))
```

Comment:
```suggestion
			w.Write([]byte("Ocorreu um erro ao renderizar a página"))
			return
```

Não faz diferença nesse caso, mas é sempre bom ter um early return em caso de erro para evitar que o código continue executando.